SU2_CFD/include/variable_structure.hpp

```diff
@@ -3638,6 +3649,10 @@ class CTurbVariable : public CVariable {
  */
 
 class CTurbSAVariable : public CTurbVariable {
+
+private:
+  su2double gamma_BC; /*!< \brief Value of the intermittency for the BC trans. model. */
```

Comment:
Any chance you will extend the approach to other turbulence models like SST? If so, we could move the gamma_BC data and getter/setter methods to the parent class. If not, this is fine.